tasks/aiida-deps-debian.yml

```diff
@@ -20,5 +25,30 @@
       - python3-psycopg2
       - python3-gi
       - python3-gi-cairo
+      - python3-venv
       - gir1.2-gtk-3.0
       - python3-tk
+
+- name: Fetch location of the Postgresql binaries
+  command: find /usr/lib/postgresql/ -name bin
+  changed_when: false
+  register: aiida_postgres_bin_raw
+  when: ansible_service_mgr != "systemd"
+
+- name: Set location of the Postgresql binaries
+  set_fact:
+    aiida_postgres_bin_location: "{{ aiida_postgres_bin_raw.stdout }}"
+  when: ansible_service_mgr != "systemd"
+
+- name: Set data location for Postgresql
+  set_fact:
+    aiida_postgres_data_location: /var/lib/postgresql/data
+  when: ansible_service_mgr != "systemd"
```

Comment:
Again this whole thing is perhaps best put into a block

Reply:
Indeed indeed. Fixed.